editor.planx.uk/src/@planx/components/PlanningConstraints/model.ts

```diff
@@ -39,13 +39,13 @@ interface Dataset {
 export const availableDatasets: Dataset[] = [
   {
     text: "Article 4 Directions",
-    val: "article4",
+    val: "articleFour",
     source: "Planning Data",
-    datasets: ["article4-direction-area"],
+    datasets: ["article-4-direction-area"],
```

Comment:
This change surprised me - was this just wrong since these options were introduced?

Reply:
Yes this was just wrong! Sorry didn't better highlight this.

These datasets are currently duplicated between API "base" metadata (actually referenced to make request) and editor modal (shown for source context only) - this is now corrected & future step here is to move to planx-core to remove duplication?

Reply:
Perfect - thanks for clarifying - this was what I figured 👍